tests/NLog.Web.AspNetCore.Tests/AspNetCoreTests.cs

```diff
@@ -222,7 +222,7 @@ public void SkipRegisterHttpContext()
             Assert.Null(webhost.Services.GetService<IHttpContextAccessor>());
         }
 
-
+        /*
```

Comment:
what happend here?

Reply:
Here something strange happened.  AppVeyor had an error message that the class MemoryTarget could not be found.  But I did not modify AspnetCoreTests as part of my PR.  I commented that method that uses the MemoryTarget so that AppVeyor would build.  I will revert that commit.

Reply:
I see what happened, I added a Target namespace to the assembly, which confused the reference Target.MemoryTarget.  I changed it to NLog.Target.MemoryTarget and it builds again.  I will work on the unit tests now.